supporting_content/dimensions/commitmentdiscounttype.md

```diff
@@ -0,0 +1,17 @@
+# Column: CommitmentDiscountType
+
+## Documentation
```

Comment:
Could we add the provider mappings and examples sections please. Many of the providers may not have this equivalent but then we can say that in the table (AWS for example has a bunch of columns for RIs and different columns for savings plans - therefore based on which ones they populate, you can tell what kind of discount it was. That's good context to provide even though its not a direct mapping for this column)

Reply:
Ok, will get some examples for this

Reply:
Microsoft has PricingModel with "SavingsPlan" and "Reservation" values.

I think AWS has a PurchaseOption column that is similar.

Reply:
GCP's equivalent here is the  "Credit Type" column

Reply:
> Microsoft has PricingModel with "SavingsPlan" and "Reservation" values.
> 
> I think AWS has a PurchaseOption column that is similar.

Would these not be for the NEW column - ie. the new, non-normalised column that is the Program name?

Reply:
Ok, please review my changes @udam-f2

Reply:
> Would these not be for the NEW column - ie. the new, non-normalised column that is the Program name?

Yes, just calling out the closest thing to the purpose of this column.